composer.json

```diff
@@ -6,10 +6,10 @@
     "license": "MIT",
     "require": {
         "php": "^8.1",
-        "cakephp/console": "5.x-dev",
+        "cakephp/console": "^5.0",
         "nette/utils": "^3.2",
-        "rector/rector": "0.19.0",
-        "symfony/string": "^6.0"
+        "rector/rector": "~0.19.0",
```

Comment:
we can try making this a version string again but lets hope rector just doesn't remove rules we are using again in a patch release...

Reply:
So far it works